addons/sale/models/sale.py

```diff
@@ -434,6 +443,15 @@ def _name_search(self, name, args=None, operator='ilike', limit=100, name_get_ui
                 return self.browse(order_ids).name_get()
         return super(SaleOrder, self)._name_search(name, args=args, operator=operator, limit=limit, name_get_uid=name_get_uid)
 
+    @api.multi
+    def update_pricelist(self):
+        if not self.env.context.get('skip_price_update'):
+            for line in self.order_line:
+                line.product_id_change()
```

Comment:
Can't we just update price for only instead of other fields(Eg, Description,UoM, account)

Reply:
@sswapnesh  Yes, right we also update other fields value...so need to change this method.
Thank you.